packages/ca_certificates.rb

```diff
@@ -123,6 +123,10 @@ def self.install
   # This isn't run from install.sh, but that's ok. This is for cleanup if updated after an install.
   def self.postinstall
     # Do not call system update-ca-certificates as that tries to update certs in /etc .
-    system "#{CREW_PREFIX}/bin/update-ca-certificates --fresh --certsconf #{CREW_PREFIX}/etc/ca-certificates.conf"
+    if File.file?("#{CREW_PREFIX}/bin/update-ca-certificates")
+      system "#{CREW_PREFIX}/bin/update-ca-certificates --fresh --certsconf #{CREW_PREFIX}/etc/ca-certificates.conf"
+    else
+      puts "#{CREW_PREFIX}/bin/update-ca-certificates is missing!".lightred
```

Comment:
Provide a suggestion how to fix too?

Reply:
Wouldn't a dependency on the package that includes the update-ca-certificates binary be the solution to fix this?

Reply:
Wait...the executable is from this package? :confused:

Reply:
Yeah that's how I realized crew upgrade was broken...